webrtc.html

```diff
@@ -2530,23 +2539,55 @@
 
           <dd>
             <p>
-              Indicates that this encoding is actively being sent.
-              Setting it to false causes this encoding to no longer be sent.
-              Setting it to true causes this encoding to be sent.
+              If true, this encoding will be sent if sufficient bandwidth if available.
+              If false, this encoding will not be sent.
             </p>
           </dd>
 
           <dt>double priority</dt>
           <dd>
             <p>
-              Indicates the relative priority of this encoding, across
-              all RtpSenders of a given PeerConnection.  When there is
+              The relative priority of this encoding across all
+              RtpSenders of a given PeerConnection.  When there is
               limited bandwidth available to a PeerConnection, higher
               prioirty encodings will be sent with more bandwidth, and
               lower priority encodings will be sent with less
               bandwidth.
             </p>
           </dd>
+
+          <dt>unsigned short payloadType</td>
+
+          <dd>
+            <p>
+              The RTP payload type being used to send media.  Setting
+              it changes the codec used to send media, but it can only
+              be set to one of the values found in
+              <code>RTCRtpParameters.codecs[n].payloadType</code>.
+            </p>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCRtpCodecParameters">
+          <dt>unsigned short payloadType</dt>
+
+          <dd>
+            <p>The RTP payload type.  This value can be set in the
+            RTCRtpParameters.encodings[0].payloadType to control which
+            codec should be used to send a given encoding.
+            </p>
+          </dd>
+
+          <dt>DOMString name</dt>
+
+          <dd>
+            <p>The name of the codec, such as "opus", "vp8", and "h264".</p>
```

Comment:
OK, it's been changed to mimeType to match https://github.com/w3c/webrtc-pc/pull/269.

At Harald's request, I also added clockRate and sdpFtmpLine.